src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -37,14 +37,16 @@ class ConsumeMessagesCommand extends Command
     private $bus;
     private $receiverLocator;
     private $logger;
+    private $defaultReceiverId;
 
-    public function __construct(MessageBusInterface $bus, ContainerInterface $receiverLocator, LoggerInterface $logger = null)
+    public function __construct(MessageBusInterface $bus, ContainerInterface $receiverLocator, LoggerInterface $logger = null, string $defaultReceiverId = null)
```

Comment:
`defaultReceiverName` (or even just `defaultReceiver`)?
The argument description mentions a name.